lib/storage/file.js

```diff
@@ -1115,7 +1119,10 @@ File.prototype.getSignedPolicy = function(options, callback) {
  * });
  */
 File.prototype.getSignedUrl = function(options, callback) {
-  if (options.expires < Math.floor(Date.now() / 1000)) {
+  var expires = new Date(options.expires).getTime();
+  var expiresInSeconds = Math.round(expires / 1000); // The API expects seconds.
+
+  if (expires < Date.now()) {
```

Comment:
Yep, you're right! I left in the `getTime()` because I need to convert it for the places it was concatenated with a string below (instead of calling Number() or `getTime()` later)... but then I remembered it needed to be seconds anyway, and forgot to update the code.

:mag: :tophat: